tests/smokeTest.js

```diff
@@ -0,0 +1,29 @@
+// smokeTest.js
```

Comment:
Inside the "Tests" directory you will have different tests such as this one "smokeTest.js". Later on, you will have e.g., "loginTest.js", "biddingTest", "searchBarTest"...
According to the page object model, inside these files, you have no actions or definition of methods; you only have to call the method (from "Pages" folder) in order for the test to be clean and readable.